src/main/webapp/WEB-INF/README.md

```diff
@@ -0,0 +1,20 @@
+## Information on the used libraries
+**Important** : All the manually-downloaded libraries which were here intially are now being installed using npm. Materialize folder is still visible here, as a download from npm, downloads all its dependencies along with the distro. This proves very costly in terms of size when war file is produced.
+
+### Libraries present:
+* `materialize`: Materialize is a modern responsive CSS framework based on Material Design by Google.
+* `backbone`: Used as the primary front-end MVC framework for this project.
+* `backbone.localstorage`: For sessions.
+* `cookie`: It is a library that helps in storing cookies.
+* `jquery`: Used by materialize and in backbone for AJAX calls and component selections.
+* `src-noconfict`: Contains ACE code editor. Used to provide an alternative to P4 file upload by providing an interface to write the P4 code on the browser.
```

Comment:
It is still there in the code. Materialize has dibs on the pre tag compared to ACE editor, hence it is not displayed. Hence, further work should override materialize defaults, to include the code editor.